README.md

```diff
@@ -8,13 +8,23 @@ Event info: https://cps-vo.org/group/ARCH/FriendlyCompetition
 
 We plan to reuse benchmarks from 2023 but feel free to propose new benchmarks or modifications to existing ones: https://github.com/verivital/ARCH-COMP2024/issues/1
 
-Specifications modified from 2023:
-
-  - Airplane
-  
-  - Double Pendulum
-
-  - Single Pendulum
+| Benchmark | Instance | Specification | Visualization | Comment |
```

Comment:
@schillic For example, the TORA benchmark has two different specifications 1 & 2. These could be mapped to a specific instance name in that column.

Reply:
But different specs always imply different instances I think.

Reply:
Yes. So for TORA it would look something like this:

| Benchmark | Instance | Specification | Network | Visualization | Comment |
|-----------|----------|---------------|-----------|---------------|---------|
| [TORA](./benchmarks/Benchmark9-Tora) | remain | Specification 1 | relu | [1,2] and [3,4] | - | 
| [TORA](./benchmarks/Benchmark9-Tora) | reach-sigmoid | Specification 2 | sigmoid | [1,2] | - | 
| [TORA](./benchmarks/Benchmark9-Tora) | reach-tanh | Specification 2 | tanh | [1,2] | - |

Reply:
Okay, but then the instance name seems sufficient to me. And a generic name like 1 and 2 does not provide more information (other than: there are two different specs here).

I guess I see the purpose of this table / README differently - which is okay. I only want a quick overview of the benchmarks and what changed compared to last year. For the rest I have to consult the report or individual files anyway.